server/app/__init__.py

```diff
@@ -1,32 +1,49 @@
 import json
 import os
-
 from werkzeug.security import generate_password_hash, check_password_hash
-from flask import Flask, render_template, jsonify, send_from_directory, request
+from flask import (
+    Flask,
+    render_template,
+    jsonify,
+    send_from_directory,
+    request,
+    redirect,
+    url_for,
+)
 from flask_sqlalchemy import SQLAlchemy
 from flask_migrate import Migrate
 
 app = Flask(__name__)
 
 # app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:////tmp/test.db"
-app.config['SLQALCHEMY_DATABASE_URI'] = 'postgresql+psycopg2://{user}:{passwd}@{host}:{port}/{table}'.format(user=os.getenv('POSTGRES_USER'),  passwd=os.getenv('POSTGRES_PASSWORD'), host=os.getenv('POSTGRES_HOST'), port=5432, table=os.getenv('POSTGRES_DB')) 
-app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
-app.secret_key = "temp string" # TODO: change this to an env variable
+app.config[
+    "SLQALCHEMY_DATABASE_URI"
+] = "postgresql+psycopg2://{user}:{passwd}@{host}:{port}/{table}".format(
+    user=os.getenv("POSTGRES_USER"),
+    passwd=os.getenv("POSTGRES_PASSWORD"),
+    host=os.getenv("POSTGRES_HOST"),
+    port=5432,
+    table=os.getenv("POSTGRES_DB"),
+)
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+app.secret_key = "temp string"  # TODO: change this to an env variable
 
 db = SQLAlchemy(app)
 migrate = Migrate(app, db)
+session = {}
+
 
 class users(db.Model):
     __tablename__ = "users"
 
     _id = db.Column(db.Integer(), primary_key=True)
     username = db.Column(db.String())
     password = db.Column(db.String())
-    # TODO: add playlists field
```

Comment:
Wouldn't we need a `playlists` attribute here too?

Something like `playlists = db.Column(db.String())`

Reply:
I left it out because we already added playlist owners to the playlists model!